src/app/speechbar/speechbar.component.html

```diff
@@ -16,25 +16,25 @@
     [class.scanHighlight]="buttonRow.isHighlighted()"
     [class.scanSelect]="buttonRow.isSelected()">
   <div class="stackedButtonContainer">
```

Comment:
These changes make the speechbar buttons (and speechbar) follow the same behaviour